core/src/main/scala/it/cwmp/utils/Loggable.scala

```diff
@@ -0,0 +1,8 @@
+package it.cwmp.utils
+
+import com.typesafe.scalalogging.Logger
+
+trait Loggable {
+
+  val logger: Logger = Logger(this.getClass.getName)
```

Comment:
A me piacerebbe vedere il nome di questa variabile cambiata in un semplice "log"
Così che le chiamate diventino log.info, log.error eccetera

Parare personale

Reply:
si può fare, accorcerebbe il codice